LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -1181,13 +1188,40 @@ function getForm(indicatorID, series) {
     $('#disabled').on("change", function(event) {
         if($(this).is(':checked'))
         {
+            $('#deleted').prop('checked', false);
+            $('#deletion-warning').css('visibility','hidden');
             $('#disabled-warning').css('visibility','visible');
         }
         else
         {
+            $('#disabled').prop('checked', false);
             $('#disabled-warning').css('visibility','hidden');
         }
     });
+    $('#deleted').keypress(function(event) {
+        if(event.keyCode === 13) {
+            event.preventDefault();
+        }
+    });
+    $('#deleted').keypress(function(e){
+        if((e.keyCode ? e.keyCode : e.which) === 13){
```

Comment:
No inception in code.

Reply:
This is currently still required to make the enter key function to toggle the selection.  It will be updated in re-write.